### ISSUE TITLE
Convert Scribe-iOS codes with translator

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-Android#10]
In reference to comments in #9, there are some tools available to translate Swift code over to Kotlin. This issue is to complete a general translation of [Scribe-iOS](https://github.com/scribe-org/Scribe-iOS/) to Kotlin. Once this is finished another issue will be opened to fine tune the translation.

Specifically [Gryphon](https://github.com/vinivendra/Gryphon) and [SwiftKotlin](https://github.com/angelolloqui/SwiftKotlin) are both mature. Initial impressions suggest that the latter should be used as there is minor support for UIKit conversions. From a basic translation the codes can then be worked on by comparing custom keyboard implementations that are also referenced in the aforementioned issue.

**Comment by andrewtavis** [comment]
The UIKit support for SwiftKotlin seems to be fairly week upon referencing [the source code for these features](https://github.com/angelolloqui/SwiftKotlin/blob/a2f50476ce78d407cddee9a52ee1b59e2957730d/Sources/SwiftKotlinFramework/plugins/UIKitTransformPlugin.swift), but then this file can be used as reference for what they should be converted to.

**Comment by andrewtavis** [comment]
SwiftKotlin was easier to get up and running and will be used for this baseline conversion.

**Comment by andrewtavis** [comment]
Closed in [470c3e5](https://github.com/scribe-org/Scribe-Android/commit/470c3e5a91459b7aac33e9d2a289c33d6fe8b3b3) :) There's some basic formatting to be done to the conversions as well, but that can also be done in #11.

**Comment by andrewtavis** [comment]
Closed in e8120cf, as 470c3e5 lacked conversions for the app itself.
